design/variable_command_design.md

```diff
@@ -0,0 +1,181 @@
+# Dockerless CLI - variable
+
+This document is the design document for the variable CLI command for the Dockerless project. The functionality for the variable CLI command already exists so all that is left is the following:
```

Comment:
The line or the sentence?

Reply:
Line itself. This is easily over 100 characters so it needs breaking down into multiple lines..

Reply:
I use [Typora](https://typora.io/) when I write MD so I can't measure how it looks in raw markdown. Is this critical because MDs are rendered in GH and won't be an excessive long line as we see when in raw